### ISSUE TITLE
opts.defaults in updated getting started not working

### ISSUE THREAD

**Issue by langehose** [ioam/holoviews#3355]
I am new to holoviews and probably my entire system setup is not sufficient. I am constantly facing issues when it comes to setting options.
My system is based on anaconda3 (python 3.7.1, param 1.8.1, holoviews, 1.10.6, bokeh 0.13.0, jupyter notebook 5.7.4) from default channel only.

First issue in Getting Started/Tabular Datasets:
overlay.opts(width=600) produces error: "TypeError: 'int' object is not iterable"

I was able to fix it for me by changing the line to
overlay.opts(plot=dict(width=600))

Just for aiding you in identifying my issue.

Even after long searches I was not able to fix the getting started on Gridded Datasets:
opts.defaults(
    opts.GridSpace(shared_xaxis=True, shared_yaxis=True),
    opts.Image(cmap='viridis', width=400, height=400),
    opts.Labels(text_color='white', text_font_size='8pt', text_align='left', text_baseline='bottom'),
    opts.Path(color='white'),
    opts.Spread(width=600),
    opts.Overlay(show_legend=False))

This is producing the error:
AttributeError: type object 'opts' has no attribute 'GridSpace'
which is the same of course for all lines. I think I could fix it by changing it back to line Magics
%opts GridSpace [shared_xaxis=True, shared_yaxis=True]
at least, this is not producing any error. However, I would like to have it jupyter independent, i.e. to use .opts() for usage in ipython/python only environments.

Of course I have imported opts, i.e. first lines of my notebook state:
import numpy as np
import holoviews as hv
from holoviews import opts
hv.extension('bokeh', 'matplotlib')

Basically I am just copy-pasting your getting started into my notebook trying to reproduce the original output.

Thank you for any helping hint
lh

**Comment by philippjfr** [comment]
``opts.defaults`` is new API introduced in 1.11.0 to replace the magics, upgrade using:

```
conda install -c pyviz holoviews

# or
pip install holoviews
```
